Lib/asyncio/base_events.py

```diff
@@ -1993,23 +2000,24 @@ def _run_once(self):
 
         # Handle 'later' callbacks that are ready.
         end_time = self.time() + self._clock_resolution
-        while self._scheduled:
-            handle = self._scheduled[0]
-            if handle._when >= end_time:
+        while scheduled:
+            when, handle = scheduled[0]
+            if when >= end_time:
                 break
-            handle = heapq.heappop(self._scheduled)
+            heapq.heappop(scheduled)
             handle._scheduled = False
-            self._ready.append(handle)
+            ready.append(handle)
```

Comment:
```suggestion
 		while scheduled and scheduled[0][0] < end_time:
            _, handle = heapq.heappop(scheduled)
            handle._scheduled = False
            ready.append(handle)
```

If you want a bit more concise loop.

Reply:
Nice, thats a bit cleaner 👍 , will adjust